examples/ecommerce/src/App.vue

```diff
@@ -97,77 +105,41 @@
-                    'hierarchicalCategories.lvl1'
-                  ]"
-                />
+                <ais-range-input attribute="price">
+                  <div slot-scope="{ currentRefinement, range, refine, canRefine }">
+                    <vue-slider
+                      :min="range.min"
+                      :max="range.max"
+                      :value="toValue(currentRefinement, range)"
+                      :disabled="!canRefine"
+                      :lazy="true"
+                      :useKeyboard="true"
+                      :enableCross="false"
+                      tooltip="always"
+                      :duration="0"
+                      @change="refine({ min: $event[0], max: $event[1] })"
+                    />
+                  </div>
+                </ais-range-input>
               </template>
             </ais-panel>
 
-            <div>
-              <ais-panel
-                :class-names="{
-                'ais-Panel-header': 'panel-header',
-              }"
-              >
-                <template slot="header">Price</template>
-
-                <template slot="default">
-                  <ais-range-input attribute="price">
-                    <div slot-scope="{ currentRefinement, range, refine, canRefine }">
-                      <vue-slider
-                        :min="range.min"
-                        :max="range.max"
-                        :value="toValue(currentRefinement, range)"
-                        :disabled="!canRefine"
-                        :lazy="true"
-                        :useKeyboard="true"
-                        :enableCross="false"
-                        tooltip="always"
-                        duration="0"
-                        @change="refine({ min: $event[0], max: $event[1] })"
-                      />
-                    </div>
-                  </ais-range-input>
-                </template>
-              </ais-panel>
-            </div>
-
-            <ais-panel
-              :class-names="{
-                'ais-Panel-header': 'panel-header',
-              }"
-            >
+            <ais-panel>
               <template slot="header">Free shipping</template>
 
               <template slot="default">
                 <ais-toggle-refinement
                   attribute="free_shipping"
                   label="Display only items with free shipping"
-                  :class-names="{
-                    'ais-ToggleRefinement-label': 'toggle-refinement-label',
-                    'ais-ToggleRefinement-checkbox': 'toggle-refinement-checkbox',
-                    'ais-ToggleRefinement-count': 'toggle-refinement-count'
-                  }"
                 />
               </template>
             </ais-panel>
 
-            <ais-panel
-              :class-names="{
-                'ais-Panel-header': 'panel-header',
-              }"
-            >
+            <ais-panel>
               <template slot="header">Rating</template>
```

Comment:
```suggestion
               <template slot="header">Ratings</template>
```

This mistake appeared before this PR.